### DISCUSSION TITLE
Memory Viewer

### DISCUSSION THREAD

**Discussion by CheMiguel23** [CheMiguel23/MemoryMesh#15] [Announcements]
**Memory Viewer** - a browser-based visualization tool for inspecting and navigating knowledge graph data. It provides an interactive interface to explore nodes, edges, and metadata with features including:

- Real-time filtering and search across all data
- Expandable metadata displays with key-value parsing
- Integrated edge visualization with clickable node navigation
- Basic statistics about nodes and edge types
- Raw JSON view for direct data inspection
- Persistent file selection and automatic reloading

![image](https://github.com/user-attachments/assets/cea53338-4ba2-4f69-9bbc-b99891705e54)

**Comment by roccho-dev** [comment]
I want to try it. 
How can we serve it? 

My usecase url -> text -> memorymesh
